lldb/source/Plugins/ABI/PowerPC/ABISysV_ppc.cpp

```diff
@@ -436,13 +436,15 @@ Status ABISysV_ppc::SetReturnValueObject(lldb::StackFrameSP &frame_sp,
       compiler_type.IsPointerType()) {
     const RegisterInfo *reg_info = reg_ctx->GetRegisterInfoByName("r3", 0);
 
-    DataExtractor data;
-    Status data_error;
-    size_t num_bytes = new_value_sp->GetData(data, data_error);
-    if (data_error.Fail())
-      return Status::FromErrorStringWithFormat(
-          "Couldn't convert return value to raw data: %s",
-          data_error.AsCString());
+    auto data_or_err = new_value_sp->GetData();
+    if (auto err = data_or_err.takeError())
+      return Status::FromError(llvm::joinErrors(
+          llvm::createStringError("Couldn't convert return value to raw data"),
```

Comment:
I think it should be this instead?

```suggestion
          llvm::createStringError("Couldn't convert return value to raw data: "),
```